site/content/integrate/slash-commands/_index.md

```diff
@@ -6,123 +6,30 @@ date: "2017-08-19T12:01:23-04:00"
 weight: 30
 aliases:
   - /integrate/other-integrations/slash-commands/
+  - /integrate/slash-commands/using-slash-commands/
 ---
 
 Slash commands are messages that begin with `/` and trigger an HTTP request to a web service that can in turn post one or more messages in response.
 
-Unlike outgoing webhooks, slash commands work in private channels and direct messages in addition to public channels, and can be configured to auto-complete when typing. Note that while Mattermost includes a number of built-in slash commands, this document concerns itself only with the slash commands that can be configured as custom integrations.
+Unlike [outgoing webhooks]({{< ref "/integrate/webhooks/outgoing" >}}), slash commands work in private channels and direct messages in addition to public channels, and can be configured to auto-complete when typing. Note that while Mattermost includes a number of [built-in slash commands]({{< ref "built-in" >}}), this document concerns itself only with the slash commands that can be configured as custom integrations.
 
+You can also create [custom slash commands]({{< ref "custom" >}}).
```

Comment:
This sentence and the last sentence paragraph above it are saying the same thing. Want to remove one?

Reply:
Good catch! Yes, I feel like the last sentence in the paragraph doesn't fit as well. I'll remove that one. :smile: